src/stores/TableStore.js

```diff
@@ -0,0 +1,75 @@
+import * as SearchUtils from '../utils/SearchUtils';
+
+export class TableStore {
+  constructor(configuration) {
+    this.CHANGE_EVENT = 'change';
+    this.eventEmitter = new EventEmitter();
+
+    this.id = configuration.id;
+    this.rows = [];
+    this.currentPage = configuration.page || 1;
+    this.totalPages = 1;
+
+    this.columnHeadings = configuration.columns;
+    this.url = configuration.dataUrl;
+    this.fetchData();
+  }
+
+  getId() {
+    return this.id;
+  }
+
+  setUrl(url) {
+    this.url = url;
+  }
+
+  getUrl() {
+    return this.url;
+  }
+
+  fetchData() {
```

Comment:
So the stores use a util or something to fetch? Or does something else fetch, and update the store using a setter?